crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -664,7 +665,11 @@ struct AEADTestVector {
         kAESKey, 16, kAESCCMCiphertext, 64, AWSLC_APPROVED, false },
 };
 
-class AEAD_ServiceIndicatorTest : public testing::TestWithParam<AEADTestVector> {};
+class AEAD_ServiceIndicatorTest : public testing::TestWithParam<AEADTestVector> {
```

Comment:
Possible to move this destructor to a base class and derive all these other algorithm-specific service indicator classes from it?